dkpro-core-opennlp-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/opennlp/OpenNlpChunker.java

```diff
@@ -55,6 +58,7 @@
 /**
  * Chunk annotator using OpenNLP.
  */
+@MimeTypeCapability(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)
```

Comment:
I would leave the `@MimeTypeCapability(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)` on the trainer and retain only the `@ResourceParameter(MimeTypes.APPLICATION_X_OPENNLP_CHUNK)` on the annotator. Doesn't that give you the desired relation?